corehq/apps/events/static/events/js/new_event.js

```diff
@@ -40,6 +40,17 @@ hqDefine("events/js/new_event", [
             'use strict';
             var self = {};
 
+            // Disable the submit button unless attendance takers are present
+            var submitBtn = $('input[type="submit"]');
```

Comment:
Isn't there a special `id` that uniquely identifies this button which can be used here? The one you are using is generic and may capture more buttons (if added in future).

Reply:
yeah you're right. I tried previously, but it didn't work. Inspecting the actual id, it looks like crispy forms do not follow the standard `id_{name}` format for this button. Anycase, I used the generated id instead, thanks for the suggestion (P.S. I saw one can actually specify what the id should be by specifying the `css_id` field on the form. Nifty)